CHANGELOG.md

```diff
@@ -24,16 +26,17 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
   - Added portuguese translation
   - Added a `database` library, that handles shared Interaction with the sql database
```

Comment:
would you mind correcting the indentation on these lines here?

Reply:
yeah sure, I corrected them once, but somehow in alll those updates I chose the wrong version